test/acceptance/features/serviceClassForward.feature

```diff
@@ -47,3 +47,110 @@ Feature: Forward Service Class into Service namespaces
                   value: v1
         """
         Then On Worker Cluster "primaza-worker", Resource "ServiceClass" with name "demo-service-sc" exists in namespace "services"
+
+    Scenario: Service Classes are pushed to new Cluster Environments' service namespaces
+        Given Primaza Cluster "primaza-main" is running
+        And   Worker Cluster "primaza-worker" for "primaza-main" is running
+        And   Clusters "primaza-main" and "primaza-worker" can communicate
+        And   On Primaza Cluster "primaza-main", Worker "primaza-worker"'s ClusterContext secret "primaza-kw" is published
+        And   On Worker Cluster "primaza-worker", service namespace "services" exists
```

Comment:
Each of these are executed in this order at the beginning of the three scenarios in this file.  This should probably be moved into the background section.